packages/shared/src/components/fields/SearchField.tsx

```diff
@@ -73,6 +81,15 @@ export const SearchField = forwardRef(function SearchField(
     setInput(null);
   };
 
+  const isPrimary = fieldType === 'primary';
+  const isSecondary = fieldType === 'secondary';
+  const SearchIcon = focused ? MagnifyingFilledIcon : MagnifyingOutlineIcon;
+  const buttonIcon = isPrimary ? (
+    <XIcon className="text-lg icon group-hover:text-theme-label-primary" />
+  ) : (
+    <ArrowIcon style={{ transform: 'rotate(90deg)' }} />
+  );
```

Comment:
Perhaps move this one out to be it's own small inline component?
Also the rotate we should have as a tailwind class out of the box right?